exercises/concept/beers/.docs/instructions.md

```diff
@@ -0,0 +1,46 @@
+# Instructions
+
+You are the new bosun of a small pirate crew that just killed their previous bosun for fowling their merriment with a bad song. As pirates would have it, your crew likes to pass the time getting loaded to the gunwales while crossing the briny deep and you are in charge of rationing the beer. The Cap'n has tasked you to minimize the number of resupply trips to make the best possible time between hauls. To achieve these conflicting requirements, and keep your crew from blowing you down when you are also loaded to the gunwales, you decide to stick to the tried-and-true 99 Bottles of Beer song using a program to help you keep the song straight and remind you to make a beer run when the ship is running low.
+
+## 1. Return the last stanza when there are 0 beers.
+
+Implement a `beerSongFrom` function that returns the last two verses of the beer song, in separate lines, when given the value `0`.
+
+> No more bottles of beer on the wall, no more bottles of beer.</br>
+> Go to shore to get some more, 99 bottles of beer on the wall!</br>
+
+## 2. Return the last two stanzas when starting with 1 beer.
```

Comment:
It's generally smoother for the student if later exercises don't change the values of earlier ones, and that once a part of the exercise is successfully completed, the tests for that part pass.

I think that asking the student to refactor the existing function will break this pattern, and mean that the tests will fail even when part 1 is done successfully, so then the student doesn't know whether they have completed part 1 or not.

Maybe this can be avoided by creating a `beerSongFrom0` in part 1, and a `beerSongFrom1` here?